tools/ibc-test/src/tests/packet.rs

```diff
@@ -0,0 +1,189 @@
+            channels.channel_id_a.value(),
+        )?;
+        let (chain_b_config, chain_b_url, chain_b_signer) = prepare_artificials(
+            &relayer.config,
+            &chains.handle_b().id(),
+            channels.channel_id_b.value(),
+        )?;
+        info!(
+            "relayer wallet balance: {} CKB on chain_a, {} CKB on chain_b",
+            wallet_balance(&rt, &chain_a_url, &chain_a_config.user_lock_script())?,
+            wallet_balance(&rt, &chain_b_url, &chain_b_config.user_lock_script())?
+        );
+
+        // 2. trigger SendPacket event on ChainA
+        info!("send send_packet transaction to chain_a");
+        let message = b"ping".to_vec();
+        let send_packet_tx = generate_send_packet_transaction(
+            &rt,
+            &chain_a_config,
+            &chain_a_url,
+            &chain_a_signer,
+            message,
+        )?;
+        let hash = send_transaction(&chain_a_url, send_packet_tx)?;
+        info!(
+            "🍻 successfully sent send_packet transaction to chain_a, hash = {}",
+            hex::encode(hash)
+        );
+
+        // 3. listen RecvPacket event on ChainB
+        info!("wait recv_packet being found on chain_b");
+        let recv_packet =
+            listen_and_wait_packet_cell(&rt, &chain_b_url, &chain_b_config, |packet| {
+                packet.is_recv_packet()
+            })?;
+        info!("🍻 successfully find recv_packet cell on chain_b");
+
+        // 4. trigger WriteAck event on ChainB
+        info!("send write_ack transaction to chain_b");
+        let acknowledgemnt = b"pong".to_vec();
+        let write_ack_tx = generate_write_ack_transaction(
+            &rt,
+            &chain_b_config,
+            &chain_b_url,
+            &chain_b_signer,
+            recv_packet,
+            acknowledgemnt,
+        )?;
+        let hash = send_transaction(&chain_b_url, write_ack_tx)?;
+        info!(
+            "🍻 successfully sent write_ack transaction to chain_b, hash = {}",
+            hex::encode(hash)
+        );
+
+        // 5. listen AckPacket event on ChainA
+        info!("wait ack_packet being found on chain_a");
+        let ack_packet =
+            listen_and_wait_packet_cell(&rt, &chain_a_url, &chain_a_config, |packet| {
+                packet.is_ack_packet()
+            })?;
+        info!("🍻 successfully find ack_packet cell on chain_a");
+
+        // 6. comsune AckPacket cell on ChainA
+        info!("send ack_packet consume transaction to chain_a");
+        let consume_ack_packet_tx = generate_consume_ack_packet_transaction(
+            &rt,
+            &chain_a_config,
+            &chain_a_url,
+            &chain_a_signer,
+            ack_packet,
+        )?;
+        let hash = send_transaction(&chain_a_url, consume_ack_packet_tx)?;
+        info!(
+            "🍻 successfully consumed ack_packet on chain_a, hash = {}",
+            hex::encode(hash)
+        );
+
+        Ok(())
+    }
+}
+
+#[ignore]
```

Comment:
Why ignore this test? Please comment the reason

Reply:
it's a specific test case that I can use to test with ease, matrix test will cover it